scripts/get-changelog.js

```diff
@@ -0,0 +1,179 @@
+// show up in generated changelogs
+const denyList = [
+  'chore(release): publish [skip ci]',
+  'chore(telemetry): update telemetry config',
+];
+
+/**
+ * Returns back the commits in an array
+ *
+ * @param {string} folder Folder to get commit log for
+ * @returns {string[]} Commits array of objects
+ */
+function getCommits(folder) {
+  const toTag = tagTo !== undefined ? tagTo : 'HEAD';
+
+  // Gets the git output between the two tags
+  const output = child
+    .execSync(
+      `git log ${tagFrom}..${toTag} --pretty=format:"%s"${delimiter} -- ${folder}`
+    )
+    .toString('utf-8');
+
+  // Generates the array of commit comments
+  return output.split(`${delimiter}\n`);
+}
+
+/**
+ * Gets the changelog content
+ *
+ * @param {string} pkgName Package name
+ * @param {string} folder Folder for git log
+ * @returns {string} Changelog content
+ */
+function getChangelog(pkgName, folder) {
+  const packageJson = require(`.${folder}/package.json`);
+  // Stores the changelog
+  let changelog = `## \`${pkgName}@${packageJson.version}\`\n`;
+
+  // Stores the list of features
+  const features = [];
+
+  // Stores the list of fixes
+  const fixes = [];
+
+  // Stores the list of chores
+  const chores = [];
+
+  const commitsInFolder = getCommits(folder);
+
+  const commitsArray = commitsInFolder.filter((commit) => {
+    for (const deny of denyList) {
+      if (commit.includes(deny)) {
+        return false;
+      }
+    }
+    return true;
+  });
+
+  commitsArray.forEach((commit) => {
+    const commitParse = commit.replace(delimiter, '');
+    if (commit.startsWith('feat')) {
+      features.push(commitParse);
+    }
+
+    if (commit.startsWith('fix')) {
+      fixes.push(commitParse);
+    }
+
+    if (
+      commit.startsWith('chore') ||
+      commit.startsWith('docs') ||
+      commit.startsWith('test') ||
+      commit.startsWith('type') ||
+      commit.startsWith('refactor')
+    ) {
+      chores.push(commitParse);
+    }
+  });
+
+  if (features.length !== 0) {
+    changelog += `### Features :rocket:\n`;
+    features.map((feature) => {
+      changelog += `- ${feature}\n`;
+    });
+    changelog += '\n';
+  }
+
+  if (fixes.length !== 0) {
+    changelog += `### Bug fixes :bug:\n`;
+    fixes.map((fix) => {
+      changelog += `- ${fix}\n`;
+    });
+    changelog += '\n';
+  }
+
+  if (chores.length !== 0) {
+    changelog += `### Housekeeping :house:\n`;
+    chores.map((chore) => {
+      changelog += `- ${chore}\n`;
+    });
+    changelog += '\n';
+  }
+
+  if (features.length === 0 && fixes.length === 0 && chores.length === 0) {
+    changelog = '';
+  }
+
+  return changelog;
+}
+
+/**
+ * Renders the log
+ */
+function generateLog() {
+  let log = '';
+
+  log += getChangelog('@carbon/ibm-products', './packages/ibm-products');
+  log += getChangelog(
+    '@carbon/ibm-products-styles',
+    './packages/ibm-products-styles'
+  );
+
+  console.log(log);
```

Comment:
Do we still want to keep this in? Or can we remove if you tested this on your fork?

Reply:
I like to keep it in, so in the case we want to run it manually from command line we can copy and paste the output from the console log